go/private/go_repository.bzl

```diff
@@ -49,10 +49,13 @@ def _go_repository_impl(ctx):
 
     # TODO(yugui): support submodule?
     # c.f. https://www.bazel.io/versions/master/docs/be/workspace.html#git_repository.init_submodules
+    _fetch_repo = "@io_bazel_rules_go_repository_tools//:bin/fetch_repo"
```

Comment:
Consider moving this logic into a function like `_maybe`. Could return a `Label`.

Also, rename to just `fetch_repo`, since this is local.

Reply:
Not sure it is worth it, there exactly two cases, and one of them (the fetch_repo one) I am actively hoping to kill this month.

The name was mostly for similarity to the gazelle rule below, where the attribute was called `_gazelle` and there is already a local variable called `gazelle` that is the stringified path, so I could not rename it